lua/gen/init.lua

```diff
@@ -54,12 +54,28 @@ end
 M.command = 'ollama run $model $prompt'
 M.model = 'mistral:instruct'
 
+local function is_serving(pname)
+    local handle, ok = pcall(io.popen, 'ps aux | grep "' .. pname .. '" | grep -v grep')
```

Comment:
https://www.lua.org/pil/8.4.html
I believe the return values are reversed for `pcall`, `ok` should be before `handle`